books.json

```diff
@@ -0,0 +1 @@
+[{"id":101,"title":"The Great Gatsby","author":"F. Scott Fitzgerald","available":true},{"id":102,"title":"To Kill a Mockingbird","author":"Harper Lee","available":false},{"id":103,"title":"Pride and Prejudice","author":"Jane Austen","available":true}]
```

Comment:
- [ ] After adding a book, the following error appears when trying to show the list of books. Kindly fix it.

![image](https://github.com/Zel-hub7/school-library/assets/62290545/e003b8c7-d6fe-42d5-83a4-0156d0be82c3)